lib/transport_web/router.ex

```diff
@@ -41,19 +41,19 @@ defmodule TransportWeb.Router do
 
     scope "/datasets" do
       get "/", DatasetController, :index
-      get "/:slug/", DatasetController, :details
+      get "/:id/", DatasetController, :details
```

Comment:
Same, this is our front's representation, it not the same context than our internal relationship with data.gouv.fr.